arch/lkl/kernel/time.c

```diff
@@ -121,7 +119,7 @@ void __init time_init(void)
 	if (ret)
 		pr_err("lkl: unable to register clocksource\n");
 
-	clockevents_config_and_register(&clockevent, HZ, 0, 0xffffffff);
+	clockevents_config_and_register(&clockevent, NSEC_PER_SEC, 1, ULONG_MAX);
```

Comment:
Thanks for the explanation. Why passing a zero clc to timer_set_oneshot can cause performance drop? Does it largely increase the number of timer ticks?